logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -849,7 +869,7 @@ public long getUnackedCount() {
         }
     }
 
-    private boolean isClosed() {
+    public boolean isClosed() {
```

Comment:
self-note: this should be private now.

Reply:
I am a little surprised that this isn't referenced from outside `Queue` (although I haven't fully-proven it, as the `JRubyAckedQueueExt#getQueue` could provide a JRuby-proxy that would map `closed?` to `isClosed()` etc).

I don't see a need to remove it from the public API; it is reasonable for a caller to inquire about this, and there is risk of missing a ruby-land call.